CONTRIBUTING.md

```diff
@@ -1,13 +1,17 @@
 # CONTRIBUTION GUIDELINES
 
 ## Before contributing
+
```

Comment:
These blank lines disappear when the markdown is rendered so they are not too vital but consistency is helpful.

Reply:
Yep, they do disappear. Anyways, this helps when
reading it in plain text (non-Markdown format). 🙂

It also helps the Markdown Lint; `CONTRIBUTING.md` had 10+ warnings.